src/style.css

```diff
@@ -0,0 +1,101 @@
+* {
+  margin: 0;
```

Comment:
not all elements have default margins and paddings, if you want to reset indentations of some particular elements, do it with each of them